dll/win32/netcfgx/netcfgx.c

```diff
@@ -439,7 +439,7 @@ InstallNetDevice(
 
     if (!LoadStringW(netcfgx_hInstance, IDS_NET_CONNECT, szBuffer, sizeof(szBuffer)/sizeof(WCHAR)))
     {
-        wcscpy(szBuffer,L"Network connection");
+        wcscpy(szBuffer,L"Network Connection");
```

Comment:
hi, since you're at it, could you add a space after the comma, before the 'L"'' Thanks!